crates/ra_ide_api/src/completion/complete_dot.rs

```diff
@@ -210,16 +212,17 @@ mod tests {
             ",
         ),
         @r###"
-       ⋮[
-       ⋮    CompletionItem {
-       ⋮        label: "the_method",
-       ⋮        source_range: [144; 144),
-       ⋮        delete: [144; 144),
-       ⋮        insert: "the_method()$0",
-       ⋮        kind: Method,
-       ⋮        detail: "fn the_method(&self)",
-       ⋮    },
-       ⋮]
+        [
+            CompletionItem {
+                label: "the_method(…)",
```

Comment:
it would maybe be nice to make the label `the_method()` if there are no parameters and `the_method(…)` if there are?

Reply:
Indeed! #1981